controller-server/src/test/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentTester.java

```diff
@@ -69,9 +69,7 @@ public DeploymentTester(ControllerTester tester) {
 
     public ApplicationController applications() { return tester.controller().applications(); }
 
-    // TODO: This thing simulates the wrong thing: the build system won't hold the jobs that are running,
-    // and so these should be consumed immediately upon triggering, and be "somewhere else" while running.
```

Comment:
I wanted to do this, and with a MockTimeline, but that'll be later (or never)...